packages/@ourworldindata/utils/src/GdocsConstants.ts

```diff
@@ -18,6 +18,7 @@ export const IMAGES_DIRECTORY = "/images/published/"
 
 export const gdocUrlRegex =
     /https:\/\/docs\.google\.com\/.+?\/d\/([-\w]+)\/?(edit)?#?/
+export const gdocIdRegex = /^[0-9A-Za-z\-]{44}$/
```

Comment:
Gdoc IDs can have underscores in them also 🙂

Reply:
Good catch :)